R/input-file.R

```diff
@@ -23,6 +23,14 @@
 #' @param buttonLabel The label used on the button. Can be text or an HTML tag
 #'   object.
 #' @param placeholder The text to show before a file has been uploaded.
+#' @param capture What source to use for capturing image, audio or video data.
+#'   This attribute facilitates user access to a device's media capture 
+#'   mechanism, such as a camera, or microphone, from within a file upload 
+#'   control.
+#'
+#'   A value of `user` indicates that the user-facing camera and/or microphone
+#'   should be used. A value of `environment` specifies that the outward-facing
+#'   camera and/or microphone should be used.
 #'
```

Comment:
```suggestion
#'
#'   By default on most phones, this will accept still photos or video. For
#'   still photos only, also use `accept="image/*"`. For video only, use
#'   `accept="video/*"`.
```